src/app/guides/how-to-get-funding-for-your-open-source-projects/how-to-get-funding-for-your-open-source-projects.mdx

```diff
@@ -0,0 +1,62 @@
+Here is a list of resources that offer aggregated sponsorships:
+
+- **Tidelift**: In addition to providing tools to help you maintain your project, Tidelift offers maintainers various ways to receive financial support.
+  For example, you can connect your [GitHub Sponsors](https://tidelift.com/subscription/how-to-connect-tidelift-with-github) account to Tidelift and get paid based on the number of sponsors you have.
+  Want to learn more? Check out their post to see how they can help you with funding: [https://support.tidelift.com/hc/en-us/articles/4406294816916](https://support.tidelift.com/hc/en-us/articles/4406294816916)
+- **Thanks.dev**: This source offers funding distribution at different levels, offers you ownership on how your donation should be used and automates as well as notifies you about your donation to other projects being used.
+  Want to learn more? Check out their FAQ post here: **[https://thanks.dev/faq](https://thanks.dev/faq)**
+- **Stackaid**: In contrast to the above two sources, Stackaid is the first sponsor to use a subscription model to help owners gain funding for their open source projects.
+  If you’re interested in learning more about how it works, check out their website here: [https://www.stackaid.us/](https://www.stackaid.us/)
+
+## Legal paperwork needed for employees contributing to open source projects
+
+Since various people are going to contribute to your project, it is important to ensure that you have the right contracts in place to ensure your project is legally protected.
+Here are some examples of legal paperwork that you can distribute to your employees:
+
+- **MIT License**:This is a very popular license for open source projects.
+  It grants permission for products and software to be modified and distributed.
+  If you want to learn more, check out this link: [https://choosealicense.com/licenses/mit/](https://choosealicense.com/licenses/mit/)
+- **Apache License 2.0**: If you want to make sure that your open source project is not modified and distributed too freely, consider using this license.
+  It encourages contributors to ask owners for their permission for their products and software to be modified and distributed.
+  If you want to learn more, check out this link: [https://choosealicense.com/licenses/apache-2.0/](https://choosealicense.com/licenses/apache-2.0/)
+- **GNU General Public License v3.0**:If you want to make sure that the source code of the software is available to anyone who receives and wants to make their own modifications and distributions, we recommend this license.
+  It ensures that your software remains free and open source.
+  If you want to learn more, check out this link: [https://choosealicense.com/licenses/gpl-3.0/](https://choosealicense.com/licenses/gpl-3.0/)
+
+## Common legal work/paperwork needed to publish projects as open source
```

Comment:
[Request] I personally have done none of these three things in most of my projects 😄. We can probably omit the section altogether. Is there a strong reason you see for keeping it in?

Reply:
Lol, you requested this information in the issue:)

Reply:
oops, where did I put this? Maybe I mispoke, or maybe I'm now disagreeing with myself 😂

Reply:
You said in the [Overview section](https://github.com/OpenContributionsProject/opencontributionssite/issues/24).

Reply:
Ah, right! Sorry 🤦 I forgot about that. Let's ignore this comment, sorry - will re-review this bit now!